create-table.md

```diff
@@ -4,87 +4,89 @@ toc: false
+## Parameters
 
+Parameter | Description
+----------|------------
+`IF NOT EXISTS` | Create a new table only if a table of the same name does not already exist in the database; if one does exist, do not return an error.<br><br>Note that `IF NOT EXISTS` checks the table name only; it does not check if an existing table has the same columns, indexes, constraints, etc., of the new table. 
+`any_name` | The name of the table to create, following these [naming rules](data-definition.html#identifiers). When the parent database is not set as the default, the name must be formatted as `database.name`.<br><br>The [`UPSERT`](upsert.html) and [`INSERT ON CONFLICT`](insert.html) statements use a temporary table called `excluded` to handle uniqueness conflicts during execution. It's therefore not recommended to use the name `excluded` for any of your tables.
+`column_def` | A comma-separated list of column definitions. Each column requires a [name](data-definition.html#identifiers) and [data type](data-types.html); optionally, a [column-level constraint](data-definition.html#column-level-constraints) can be specified. Column names must be unique within the table but can have the same name as indexes or constraints.<br><br>Any `PRIMARY KEY`, `UNIQUE`, and `CHECK` [constraints](data-definition.<html id="constraints"></html>) defined at the column level are moved to the table level as part of the table's creation. Use the `SHOW CREATE TABLE` statement to view them at the table level.
+`index_def` | An optional, comma-separated list of [index definitions](data-definition.html#indexes). For each index, the column(s) to index must be specified; optionally, a name can be specified. Index names must be unique within the table but can have the same name as columns or constraints. See the [Create Indexes with a Table](#create-indexes-with-a-table) example below.<br><br>The [`CREATE INDEX`](create-index.html) statement can be used to create an index separate from table creation.  
+`table_constraint` | An optional, comma-separated list of [table-level constraints](data-definition.html#table-level-constraints). Constraint names must be unique within the table but can have the same name as columns or indexes.
 
 ## Examples
 
-If the table does **not** have a `PRIMARY KEY` constraint defined, a mandatory column called `rowid` of type `INT` will be added and automatically populated with a unique id that will be used as the primary key. The column will be added even if the table has an `UNIQUE` constraint.
+### Create a Table With a Primary Key
+
+In this example, we create a table with three columns. One column is given the [`PRIMARY KEY`](data-definition.html#primary-key) constraint, another is given the [`UNIQUE`](data-definition.html#unique) constraint, and the third has no constraints. Indexes are automatically created for columns with the `PRIMARY KEY` and `UNIQUE` constraints.
```

Comment:
I think "Indexes are automatically created for columns with the `PRIMARY KEY`" is a little bit misleading -- an index isn't *created* for `PRIMARY KEY` really, and wording it this way may reinforce the (erroneous) idea that omitting it somehow saves on space or indexing costs.

Maybe something like "as a result of how CockroachDB stores rows, the  column(s) of the PRIMARY KEY are automatically indexed" or something to that effect might avoid the confusion?